libs/net/models/Areas/Subscriber/Models/User/UserModel.cs

```diff
@@ -0,0 +1,104 @@
+using TNO.Entities;
+
+namespace TNO.API.Areas.Subscriber.Models;
+
+/// <summary>
+/// UserModel class, provides a model that represents an user.
+/// </summary>
+public class UserModel
```

Comment:
Subscribers should not need this

Reply:
On the content model it is used for getting the owner, is this something we want to keep?

Reply:
Subscriber's don't care which editor currently owns the content.  It's a feature we built in to enable assigning and identifying who worked on content.